File: TransCoder_tokenized_test_set_functions/cpp/REMOVE_CONSECUTIVE_DUPLICATES_STRING.cpp
void removeDuplicates ( char S [ ] ) {
  int n = strlen ( S );
  if ( n < 2 ) return;
  int j = 0;
  for ( int i = 1;
  i < n;
  i ++ ) {
    if ( S [ j ] != S [ i ] ) {
      j ++;
      S [ j ] = S [ i ];
    }
  }
  j ++;
  S [ j ] = '\0';
}

File: results/java_codex/REMOVE_CONSECUTIVE_DUPLICATES_STRING.java
void removeDuplicates(char S[]) {
  int n = sizeof(S);
  if (n < 2) {
    return;
  }
  int j = 0;
  for (int i = 1; i < n; i++) {
    if (S[j] != S[i]) {
      j++;
      S[j] = S[i];
    }
  }
  cout << S[0, j + 1];
}